scripts/gendocs/templates/members.tpl

```diff
@@ -4,7 +4,8 @@
 {{ if not (hiddenMember .)}}
 <tr>
     <td>
-        <code>{{ fieldName . }}</code><br/>
+        <code>{{ fieldName . }}</code>
+        <br/>
```

Comment:
note: I made a few layout changes like this which helped me to reason about whether the template was broken or not. I think it's slightly cleaner to read.